src/promptflow/promptflow/_sdk/_submitter/experiment_orchestrator.py

```diff
@@ -127,91 +158,542 @@ def _test_flow_node(self, node, test_context):
     def _test_command_node(self, *args, **kwargs):
         raise NotImplementedError
 
-    def start(self, experiment: Experiment, **kwargs):
-        """Start an experiment.
+    def start(self, nodes=None, from_nodes=None):
+        """Start an execution of an experiment.
+
+        Start an orchestrator to schedule node execution according to topological ordering.
 
-        :param experiment: Experiment to start.
-        :type experiment: ~promptflow.entities.Experiment
-        :param kwargs: Keyword arguments.
-        :type kwargs: Any
+        :param nodes: Nodes to be executed.
+        :type nodes: list
+        :param from_nodes: The branches in experiment to be executed.
+        :type from_nodes: list
+        :return: Experiment info.
+        :rtype: ~promptflow.entities.Experiment
         """
         # Start experiment
         logger.info(f"Starting experiment {experiment.name}.")
-        context = ExperimentTemplateContext(experiment)
         experiment.status = ExperimentStatus.IN_PROGRESS
         experiment.last_start_time = datetime.utcnow().isoformat()
         experiment.last_end_time = None
         self.experiment_operations.create_or_update(experiment)
-        # Ensure nodes order
-        resolved_nodes = ExperimentHelper.resolve_nodes_to_execute(experiment)
+        self._update_orchestrator_record(status=ExperimentStatus.IN_PROGRESS, pid=os.getpid())
+        self._start_orchestrator(nodes=nodes, from_nodes=from_nodes)
+
+    def async_start(self, executable_path=None, nodes=None, from_nodes=None):
+        """Start an asynchronous execution of an experiment.
+
+        :param executable_path: Python path when executing the experiment.
+        :type executable_path: str
+        :param nodes: Nodes to be executed.
+        :type nodes: list
+        :param from_nodes: The branches in experiment to be executed.
+        :type from_nodes: list
+        :return: Experiment info.
+        :rtype: ~promptflow.entities.Experiment
+        """
+        logger.info(f"Queuing experiment {self.experiment.name}.")
+        self._update_orchestrator_record(status=ExperimentStatus.QUEUING)
+
+        executable_path = executable_path or sys.executable
+        args = [executable_path, __file__, "start", "--experiment", self.experiment.name]
+        if nodes:
+            args = args + ["--nodes"] + nodes
+        if from_nodes:
+            args = args + ["--from-nodes"] + from_nodes
+        # Start an orchestrator process using detach mode
+        logger.debug(f"Start experiment {self.experiment.name} in background.")
+        self._start_process_in_background(args, executable_path)
+        return self.experiment
+
+    def _start_process_in_background(self, args, executable_path=None):
+        if platform.system() == "Windows":
+            os.spawnve(os.P_DETACH, executable_path, args, os.environ)
+        else:
+            subprocess.Popen(" ".join(["nohup"] + args + ["&"]), shell=True, env=os.environ)
```

Comment:
not sure if Configuration could pass to sub processes.
for examples we could change HOME PATH.

Reply:
For now, only environment variables will be passed to subproces. There shouldn't be any other parameters that need to be configured for start experiment at the moment, as the executor should switch to the snapshot folder when node is executed

Reply:
offline sync. Currently, the experiment is started through the CLI, and the global configuration can only be configured through pf config. There will be support for sdk passing config in a new PR